src/main/java/org/mskcc/sequencer/SequencerDoneController.java

```diff
@@ -105,6 +105,16 @@ protected boolean sampleIsIGOComplete(String sample, List<ArchivedFastq> fastqs,
         return false;
     }
 
+    @GetMapping(value = "/fastqsbyigoid/{igoid}")
+    public List<ArchivedFastq> findFastqsByIGOId(@PathVariable String igoid) {
+        log.info("/fastqsbyigoid/" + igoid);
+        if (igoid == null || igoid.length() < 7)
+            return null;
```

Comment:
Should this be an empty list?

Reply:
Could be, I think since LIMSRest is the primary consumer either way is okay.